source/scss/_pf-basic-variables.scss

```diff
@@ -491,3 +502,19 @@ $state-danger-border:            $brand-danger;
 $alert-padding-x:             $spacer;
 $alert-link-font-weight:      normal;
 $pf-alert-icon-font-size:     $icon-size-sm;
+
+
+// Tabs
+//
+// Define tab colors, etc.
+
+$nav-tabs-border-color:                       $color-pf-black-200;
+// $nav-tabs-border-width:                       $border-width !default;
+// $nav-tabs-border-radius:                      $border-radius !default;
+$nav-tabs-link-hover-border-color:            $color-pf-black-200;
```

Comment:
Please review this. The hover appears to be dark blue instead of dark black.

https://srambach.github.io/patternfly-atomic/?p=basics-tabs-variations